Xamarin.Forms.Core/Shell/ShellSection.cs

```diff
@@ -8,7 +8,12 @@
 
 namespace Xamarin.Forms
 {
-	[ContentProperty("Items")]
+	public class ShellTab : ShellSection
```

Comment:
Can we do `Tab` instead?

Reply:
if the mid-term goal is to deprecate NavigationPage, TabbedPage, and MDP, I'm fine if we have some overlap in names. otherwise, it's better to avoid confusion